application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -127,6 +140,26 @@ public void onButtonClick(ButtonInteractionEvent event, List<String> args) {
         event.editMessage(MessageEditData.fromCreateData(message)).queue();
     }
 
+    @Override
+    public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {
+        AutoCompleteQuery focusedOption = event.getFocusedOption();
+
+        if (!focusedOption.getName().equals(CANCEL_REMINDER_OPTION)) {
+            throw new IllegalArgumentException(
```

Comment:
Wrong exception. Should be `AssertionError`, as it would indicate a bug in the code.

Reply:
iirc, i got it from some other place in the codebase, which had similar case

Reply:
then id flag this other place as well lol